deploy/postgres/ansible/check-postgresql-status.sh

```
@@ -0,0 +1,128 @@
+echo "PostgreSQL packages: $(dpkg -l | grep postgresql | wc -l) found"
+echo "PostgreSQL processes: $(ps aux | grep postgres | grep -v grep | wc -l) running"
+echo "PostgreSQL config dirs: $(find /etc -name "*postgresql*" -type d 2>/dev/null | wc -l) found"
+echo "PostgreSQL data dirs: $(find /var/lib -name "*postgresql*" -type d 2>/dev/null | wc -l) found"
+echo "Relevant mount points: $(mount | grep -E "(srv|data|postgresql)" | wc -l) found"
+echo ""
+if systemctl is-active postgresql >/dev/null 2>&1; then
+    echo "PostgreSQL service: ACTIVE"
+else
+    echo "PostgreSQL service: INACTIVE"
+fi
+' 2>/dev/null
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)